example/sparse/linear_classification.py

```diff
@@ -78,18 +78,18 @@ def next(self):
 
 datasets = { 'kdda' : kdda, 'avazu' : avazu }
 
-def regression_model(feature_dim):
-     initializer = mx.initializer.Normal()
+def linear_model(feature_dim):
      x = mx.symbol.Variable("data", stype='csr')
      norm_init = mx.initializer.Normal(sigma=0.01)
-     v = mx.symbol.Variable("v", shape=(feature_dim, 1), init=norm_init, stype='row_sparse')
-     embed = mx.symbol.dot(x, v)
+     weight = mx.symbol.Variable("weight", shape=(feature_dim, 1), init=norm_init, stype='row_sparse')
```

Comment:
Yes, we are aware that the dimension could be 64 bit and we're trying to fix that before 1.0 release  https://github.com/apache/incubator-mxnet/issues/7319